### DISCUSSION TITLE
[Feature Request] Update All button

### DISCUSSION THREAD

**Discussion by xeome** [louislam/dockge#512] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Currently, Dockge provides an easy-to-use interface for managing individual Docker Compose files, including the ability to update each file individually. However, there is no built-in functionality to update all Docker Compose files at once.

I believe Adding an "Update All" button to the Dockge interface would be very beneficial. This button would allow users to update all Docker Compose files managed by Dockge with a single click

### ✔️ Solution

An "Update All" button in the Dockge interface that, when clicked, triggers an update for all Docker Compose files currently managed by Dockge.

The feature should include logic to avoid trying to update containers that are already up-to-date.

**Comment by InnocentRain** [comment]
This is something i would also love to see 👍

**Reply by DomiiBunn** [reply]
If I remember and have the time I'll build a public docker imagine with the button

**Reply by DomiiBunn** [reply]
You can try. `registry.gitlab.com/domiibunn/dockge:latest` it's the build from #576 

⚠️Keep in mind I am not commiting to keep it up to date any further than said PR⚠️

**Comment by cyril59310** [comment]
This feature is already in progress in the PR #576
